src/zabapgit_js_common.w3mi.xml

```diff
@@ -15,13 +15,13 @@
      <RELID>MI</RELID>
      <OBJID>ZABAPGIT_JS_COMMON</OBJID>
      <NAME>filename</NAME>
-     <VALUE>common.js</VALUE>
+     <VALUE>~wwwtmp.js</VALUE>
     </WWWPARAMS>
     <WWWPARAMS>
      <RELID>MI</RELID>
      <OBJID>ZABAPGIT_JS_COMMON</OBJID>
      <NAME>mimetype</NAME>
-     <VALUE>text/javascript</VALUE>
+     <VALUE>application/javascript</VALUE>
```

Comment:
Maybe we should exclude this change?

Reply:
as above.
It's a side effect of editing the js file